cmd/serverless/main.go

```diff
@@ -337,6 +337,8 @@ func runAgent(stopCh chan struct{}) (serverlessDaemon *daemon.Daemon, err error)
 		)
 	}
 
+	serverlessDaemon.ComputeGlobalTags(configUtils.GetConfiguredTags(config.Datadog, true))
```

Comment:
we can compute global tags in `main.go` now (before handling our first invocation) because we now have access to the functionArn through extension registration: https://github.com/DataDog/datadog-agent/pull/18748